code/__DEFINES/atmospherics/atmos_helpers.dm

```diff
@@ -99,12 +99,17 @@ GLOBAL_LIST_INIT(atmos_adjacent_savings, list(0,0))
  *
  * Not immediately obvious, but saves us operation time.
  * 
+ * We put a lot of parentheses here because the numbers get really really big. 
+ * By prioritizing the division we try and tone down the number so we dont get overflows.
+ * 
  * Arguments:
- * * temperature_delta: T2 - T1. [/datum/gas_mixture/var/temperature]
+ * * temperature_delta: T2 - T1. [/datum/gas_mixture/var/temperature]. 
+ * If you have any moderating (less than 1) coefficients and are dealing with very big numbers 
+ * multiply the temperature_delta by it first before passing so we get even more breathing room.
  * * heat_capacity_one:  gasmix one's [/datum/gas_mixture/proc/heat_capacity]
  * * heat_capacity_two: gasmix two's [/datum/gas_mixture/proc/heat_capacity]
  * Returns: The energy gained by gas mixture one. Negative if gas mixture one loses energy.
  * Honestly the heat capacity is interchangeable, just make sure the delta is right.
  */
 #define CALCULATE_CONDUCTION_ENERGY(temperature_delta, heat_capacity_one, heat_capacity_two)\
-	(temperature_delta * heat_capacity_one * heat_capacity_two / (heat_capacity_one+heat_capacity_two))
+	(temperature_delta * (heat_capacity_one * (heat_capacity_two / (heat_capacity_one+heat_capacity_two))))
```

Comment:
```suggestion
	((temperature_delta) * ((heat_capacity_one) * ((heat_capacity_two) / ((heat_capacity_one) + (heat_capacity_two)))))
```
shoulda done this before, basically if someone put x + y into temperature_delta you'd get unexpected behavior
forgot to mention it, les do it now